tests/foreman/cli/test_contentcredentials.py

```diff
@@ -506,27 +507,29 @@ def test_positive_update_key_for_empty_product(module_org):
     product = make_product({'organization-id': module_org.id})
     gpg_key = make_content_credential({'organization-id': module_org.id})
     # Associate gpg key with a product
-    Product.update(
+    target_sat.cli.Product.update(
         {'gpg-key-id': gpg_key['id'], 'id': product['id'], 'organization-id': module_org.id}
     )
     # Verify gpg key was associated
-    product = Product.info({'id': product['id'], 'organization-id': module_org.id})
+    product = target_sat.cli.Product.info({'id': product['id'], 'organization-id': module_org.id})
     assert product['gpg']['gpg-key'] == gpg_key['name']
     # Update the gpg key
     new_name = gen_choice(list(valid_data_list().values()))
-    ContentCredential.update(
+    target_sat.cli.ContentCredential.update(
         {'name': gpg_key['name'], 'new-name': new_name, 'organization-id': module_org.id}
     )
     # Verify changes are reflected in the gpg key
-    gpg_key = ContentCredential.info({'id': gpg_key['id'], 'organization-id': module_org.id})
+    gpg_key = target_sat.cli.ContentCredential.info(
+        {'id': gpg_key['id'], 'organization-id': module_org.id}
+    )
     assert gpg_key['name'] == new_name
     # Verify changes are reflected in the product
-    product = Product.info({'id': product['id'], 'organization-id': module_org.id})
+    product = target_sat.cli.Product.info({'id': product['id'], 'organization-id': module_org.id})
     assert product['gpg']['gpg-key'] == new_name
 
 
 @pytest.mark.tier2
-def test_positive_update_key_for_product_with_repo(module_org):
+def test_positive_update_key_for_product_with_repo(module_target_sat, module_org):
```

Comment:
why using module_target_sat here instead of target_sat? can you keep it consistent?